src/DonationForms/V2/resources/components/DonationFormsListTable.tsx

```diff
@@ -330,6 +352,9 @@ export default function DonationFormsListTable() {
                         </a>
                     </>
                 )}
+                {state.showDefaultFormTooltip && (
```

Comment:
@alaca We need to use the `isCampaignDetailsPage` here as well to prevent displaying the tooltip on the All Forms page as you can check on the attached screenshot. 

![image](https://github.com/user-attachments/assets/d8c5524a-1d02-48a4-8ca2-ef14771ff986)

Reply:
Good catch!

Reply:
Actually, there is the possibility of creating a feature flag that allows old customers to keep this page visible.

Reply:
Resolved https://github.com/impress-org/givewp/pull/7590/commits/dc269feb19ad8ddc593f7de3aec581bac26b217d